TL improvement (#526)
07th-mod/higurashi-console-arcs: Update/omot_005_choice4.txt

//r…問題は、いつ彼に真相を打ち明けるかだろう。いきなり話したところで、果たしてまじめに耳を傾けてくれるかどうか、…まだ、自信が持てない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…問題は、いつ彼に真相を打ち明けるかだろう。いきなり話したところで、果たしてまじめに耳を傾けてくれるかどうか、…まだ、自信が持てない。",

> 			NULL, "...The problem is, when should I divulge the truth to him? Even if I were to suddenly tell him about this... I'm still not confident he would take it seriously.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...The question is, when should I divulge the truth to him? Even if I were to suddenly tell him about this... I'm still not confident he would take it seriously.", Line_Normal);
```

//rにこにこと、まるで人を見下したように明るい笑顔にぞわぞわっ、と嫌悪感が駆け巡ったが、すぐに気を取り直していつもの『にぱー』笑いで応えた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "にこにこと、まるで人を見下したように明るい笑顔にぞわぞわっ、と嫌悪感が駆け巡ったが、すぐに気を取り直していつもの『にぱー』笑いで応えた。",

> 	ClearMessage();

Reviewer: ```suggestion
			NULL, "I felt completely disgusted when she looked down at me with a bright smile, but I quickly regained my composure and responded with my usual smile and a \"Nipah.\"", Line_Normal);
	ClearMessage();
```

//r…わずかに視界の片隅に、入江の悲しそうな顔が見えたが、あえて無視した。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…わずかに視界の片隅に、入江の悲しそうな顔が見えたが、あえて無視した。",

> 			NULL, "...I sensed a hint of sadness on Irie's face in the corner of my view, but I purposely ignored it.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...I sensed a hint of sadness on Irie's face in the corner of my view, but I did my best to ignore it.", Line_Normal);
```
